### ISSUE TITLE
Easy user switching

### ISSUE THREAD

**Issue by Suncatcher** [wallabag/wallabagger#79]
This feature-request is closely connected to [my request](#2989) to main wallabag.
While it is still useful to have mass moving on from the admin UI, it is also cool to add individual articles to certain users on-demand.
What I propose is adding multiple user to Wallbagger settings UI and easy switching between them.
Now we have only single user credentials
![image](https://cloud.githubusercontent.com/assets/6388034/25577757/747dc880-2e71-11e7-8061-57782c5dc16f.png)

It would be nice to add multiple users here which can be switched, e.g. via radiogroup or another way.

**Comment by Simounet** [comment]
Hi there,
If I understood it well, you want to push articles to various wallabag's users of a same instance?

**Comment by Suncatcher** [comment]
Yes, exactly.

**Comment by Simounet** [comment]
Ok, I have to look at the current code but I think we might:
- add a field to add users' credentials on the addon preferences' page
- if there is more than one user, clicking on the wallabagger icon must let the user choose to which user the content sent
- add the user's list into the context-menu

**Comment by Suncatcher** [comment]
Yes, the latter two options would be more handy. 
For example, if there is more than one credential entered, then on click to Wallabagger icon, the user is prompted to choose the target user. If there is a single one, then bagging goes as usual.

**Comment by Simounet** [comment]
Yep. We will think to implement this after your request's approval on wallabag.
